libkbfs/conflict_resolver.go

```diff
@@ -1674,14 +1676,10 @@ func (cr *ConflictResolver) getActionsToMerge(unmergedChains *crChains,
 // directory (i.e., files are merged directly, they are just
 // renamed/copied).  It also collapses each action list to get rid of
 // redundant actions.
-func collapseActions(unmergedChains *crChains,
+func collapseActions(unmergedChains *crChains, unmergedPaths []path,
 	mergedPaths map[BlockPointer]path,
-	actionMap map[BlockPointer]crActionList) {
+	actionMap map[BlockPointer]crActionList) (newUnmergedPaths []path) {
```

Comment:
`newUnmergedPaths` sounds like it's supposed to replace `unmergedPaths`, but it looks like it's a list of additional paths. Worth a comment and/or better name, I think.